pkg/config/types.go

```diff
@@ -179,4 +179,16 @@ type Config struct {
 	DynamicFilePath string
 	//use UserId for mapping, IdentityArn is not used any more when DynamicFileUserIDStrict=true
 	DynamicFileUserIDStrict bool
+	//reserved prefix config for username
+	ReservedPrefixConfig map[string]ReservedPrefixConfig
+}
+
+type ReservedPrefixConfig struct {
+	//backend mode defined in line 171
```

Comment:
Nit: line numbers change often, maybe just refer to the type and field, like `Config.BackendMode`

Same comment on space between comment and text